.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,16 +10,18 @@ Thanks for stopping by to let us know something could be better!
 
 Please run down the following list and make sure you've tried the usual "quick fixes":
 
-  - Search the issues already opened: https://github.com/googleapis//issues
+  - Search the issues already opened: https://github.com/googleapis/java-bigtable/issues
   - Check for answers on StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform
 
 If you are still having issues, please include as much information as possible:
 
 #### Environment details
 
-1. OS type and version:
-2. Java version:
-3.  version(s):
+1. Specify the API at the beginning of the title. For example, "BigQuery: ...").
```

Comment:
This is unfortunate :(

Reply:
This is in the upstream templates. We can fix it there: https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/java_library